lib/shared/src/models/sync.ts

```diff
@@ -192,7 +192,8 @@ export function syncModels({
                                         ),
                                         enableToolCody,
                                         featureFlagProvider.evaluateFeatureFlag(
-                                            FeatureFlag.EnhancedContextWindow
+                                            FeatureFlag.EnhancedContextWindow,
+                                            true
```

Comment:
nit: Is this boolean for the "force refresh" value? If so, could we add a comment, makes it easier to read what these primitive types mean... If this pattern is okay :) 

```suggestion
                                            true /** force refresh */
```